wallet/keychain.go

```diff
@@ -34,6 +35,26 @@ func (k *HmacKey) FromHexString(hexString string) (HmacKey, error) {
 	return *k, nil
 }
 
+// ToBase64String converts the HMAC key to a base64 string
+func (k *HmacKey) ToBase64String() string {
+	return base64.StdEncoding.EncodeToString(k[:])
+}
+
+// FromBase64String converts a base64 string to an HMAC key
+func (k *HmacKey) FromBase64String(b64String string) (HmacKey, error) {
+	bytes, err := base64.StdEncoding.DecodeString(b64String)
+	if err != nil {
+		return HmacKey{}, err
```

Comment:
nit: should we be returning `nil, err` here and below?

Reply:
this method returns a value type, so `nil` cannot be used